func/HeroicBashLauncher.py

```diff
@@ -1,50 +1,79 @@
+#!/bin/env python3
 #Main file that takes bash arguments
 
-import os,sys
+import os, sys, argparse
 import configpath
 from frombash import frombash
 from listinstalled import listinstalled
 from steam import createscript, addtosteam
 from flatpak import launchflatpakgame
+from zenity import zenity_popup, zenity_enable
 
 #Print current version
 print("Using Bash Launcher 2.4.2\n")
 
-#Check if Zenity is installed
-print("Checking if Zenity is installed:")
-checkzenity = os.system('zenity --version')
+parser = argparse.ArgumentParser(description='Create launcher scripts from games installed with Heroic')
+parser.add_argument("--zenity", action="store_true",
+                    help="Enable zenity popups showing status as well as printing to stdout")
+group = parser.add_mutually_exclusive_group()
+group.add_argument("--create_all", action="store_true",
+                    help="List installed Heroic games, create launch files, and add them all to Steam")
+group.add_argument("--create", metavar="gamename", type=str,
```

Comment:
This is more like "Only add games to Steam" and should be "--add". This refers to the code from lines 50-60 (AddToSteam functionality) where the game's name is the only argument and the program proceeds to add the game to Steam. It doesn't create the launch file.

The metavar should be " 'gamename' ". This corresponds to the syntax of the argument used in AddToSteam.sh.

Plus, the program is proceeding to create all game launch scripts whenever I use '--create Inside' and does not perform the expected functionality.